src/towncrier/create.py

```diff
@@ -32,30 +35,38 @@
 )
 @click.option(
     "--edit/--no-edit",
-    default=False,
+    default=None,
     help="Open an editor for writing the newsfragment content.",
-)  # TODO: default should be true
+)
 @click.option(
     "-c",
     "--content",
     type=str,
-    default="Add your info here",
+    default=DEFAULT_CONTENT,
     help="Sets the content of the new fragment.",
 )
-@click.argument("filename")
+@click.option(
+    "--eof-newline/--no-eof-newline",
+    default=None,
```

Comment:
Can we have the default value defined here?

```suggestion
    default=False,
```

Reply:
No, since the default of `None` allows falling back to using the configuration value. Setting `True` or `False` allows a user to always override that configuration default.

Reply:
Do we need this as a command line option?

Can we just go with the config file ?

Reply:
Yeah, it's a pretty obscure setting to ever need to change.